src/twig.functions.js

```diff
@@ -232,6 +236,57 @@ var Twig = (function (Twig) {
             }
 
             return getRandomNumber(LIMIT_INT31-1);
+        },
+
+        /**
+         * Returns the content of a template without rendering it
+         * @param {string} name
+         * @param {boolean} [ignore_missing=true]
+         * @returns {string}
+         */
+        source: function(name, ignore_missing) {
+            var templateSource;
+            var templateFound = false;
+            var options = {
+                id: 'templates/source.twig',
+                method: 'ajax',
+                async: false,
+                fetchTemplateSource: true
+            };
+
+            //default ignore_missing to true
```

Comment:
I think, in Twig.php ignore_missing defaults to false? Also, it throws an exception ...